test/lib/index.test.ts

```diff
@@ -15,6 +19,19 @@ describe('issuePathToLineNumber', () => {
       );
     }).toThrowError('Unknown format');
   });
+
+  it('returns first node if it can not match the path ', () => {
```

Comment:
I'm hazy on the context of this PR - ultimately, do we not want to return some semantically invalid value (e.g. -1) when the path can't be matched, so that clients can then choose not to do things with it (e.g. show a code preview)?

Perhaps clients will break if we jump straight to that, and this is a first step? This is all speculation on my part.

Reply:
yes you are right. This PR could have just been the fist commit with the added tag. 

I added this test anyway because it helped me debug, even though it will most probably be updated/removed in the next one to return something else when we can't find the path.